discovery-infra/tests/conftest.py

```diff
@@ -100,6 +115,15 @@ def get_api_client(offline_token=env_variables['offline_token'], **kwargs):
     return assisted_service_api.create_client(url, offline_token, **kwargs)
 
 
+def get_available_openshift_versions() -> List[str]:
+    available_versions = list(get_api_client().get_openshift_versions().keys())
+    specific_version = env_variables["openshift_version"]
+    if specific_version and specific_version in available_versions:
```

Comment:
Maybe if specific version doesn't much what available we should raise
Doesn't seem right to keep going with what we have